### DISCUSSION TITLE
Option to have Sub-packs be collapsed when opening a shop pack

### DISCUSSION THREAD

**Discussion by NyViking** [gameserverapp/Platform#251] [Suggestions]
As the title says, having a option to have the sub-packs be collapsed when opening a shop pack would be nice.
Currently making a shop pack for ARK, and adding in the Land based ones with a randomizer(4x25% for different skins), after about 30+ entries the site takes a chunk to load between each saves. Just thought it would be a nice quality of life addition.

**Comment by Decollates** [comment]
Signalboosting this.
